### DISCUSSION TITLE
Why does await request.formData(), work only on edge?

### DISCUSSION THREAD

**Discussion by alexnguyen98** [vercel/next.js#68106] [App Router]
When i tried `await request.formData()` locally it worked correctly, but when i tried on production it kept saying it could not parse FormData. Only after I changed the runtime into edge it worked. The documentation is not clear and transparent about those requirements.

**Comment by icyJoseph** [comment]
It should work fine in Node.js though 🤔 

What Node.js version do you have in your deployed version?

**Reply by alexnguyen98** [reply]
The newest one

**Reply by icyJoseph** [reply]
Exact number?
